docs/wdc_webengine.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Troubleshoot WDC WebEngine Issues in Tableau 2019.4 and later
+layout: docs
+---
+Prior to Tableau 2019.4, Web Data Connectors used an internal browser in Tableau called Qt WebKit. In Tableau 2019.4 and later, Web Data Connectors use Qt WebEngine (a Chromium-based browser). Qt WebEngine is a significant upgrade over previous browser and WDC developers will benefit from this upgrade with the ability to use modern JavaScript, and Chrome developer tools for debugging WDC code. However,
+some customers might experience problems using their existing connectors in Tableau 2019.4 (and later) because of the change from Qt WebKit to Qt WebEngine in Tableau. This topic describes some of these issues and provides possible solutions.
+
+---
+
+* TOC
+{:toc}
+
+---
+
+### Before you begin
+
+Before diagnosing problems, be sure to launch Tableau Desktop with debugging enabled  (`--remote-debugging-port=9000` option) and check for console messages. These will also be written to the Tableau log file (regular log file on Tableau Desktop; the `tabprotosrv` logs on Tableau Server).
+
+### `tableau.exe -DebugWDC` doesn't work
+
+**Resolution:** Under WebEngine, we use regular Chrome to debug WDCs. Launch Tableau with `--remote-debugging-port=9000` in order to debug it. You no longer need to launch an additional window for debugging.
+
+### Chrome debugging doesn't work when refreshing the extract
+
+**Resolution:** Developers might find it challenging staying "connected" to a remote debugging session in Chrome when performing refreshes. This is because Tableau disconnects the connection before performing a refresh of an extract, causing the Chromium debugging session to be reset and a new Chromium page instance used in the new connection. The recommended way to debug getSchema and getData are through the Tableau data pane: after dragging out a table to the join area, connect to Chrome debug tools, set your breakpoints, and then click "Update Now".
```

Comment:
... and a new Chromium page instance **to be** used?

Reply:
I reworked this section. 

"This is because Tableau disconnects the debugging connection before performing a refresh of an extract, causing the debugging session to be reset, which requires that you open a new instance of the Chrome Developer Tools to make the new connection. "